argocd/argocd/values.yaml

```diff
@@ -36,6 +36,14 @@ argo-cd:
     metrics:
       serviceMonitor:
         enabled: true
+    volumeMounts:
+      - mountPath: /etc/ssl/certs/ca.crt
+        name: certificate
+        subPath: ca.crt
+    volumes:
+    - name: certificate
+      secret:
+        secretName: argocd-tls
```

Comment:
Are you sure this works?
I tested this on my side and it does not work for me because the secret generated by cert-manager have a random suffix.

Reply:
Works fine for me. The secret name is indeed suffixed when created but it is recreated after a couple of seconds and the suffix is omitted.

Reply:
OK, This is weird, ArgoCD indeed starts... Anyway, can you adopt the same YAML format convention regarding array indentation? you use 2sp in `volumeMounts` and 0sp in `volumes`.
My linter yield when there is no space before array, so I personally prefer having 2sp like you did in `volumeMounts`.
When trying to log into ArgoCD with OIDC I get: `Failed to query provider "https://keycloak.apps.172-17-0-8.nip.io/auth/realms/kubernetes": 404 Not Found: {"error":"Realm does not exist"}`, but I guess it's not related to this change. We must have a race condition again...

Reply:
That could be caused by an attempt to log in via OIDC before the creation of the realm. Not sure though..
On the other hand, I also saw a couple of errors in the operator's logs although all the resources are created (realm, user...).
I also had a "client no found" while trying to log in via OIDC a couple of times, not sure if this is linked to #260.
I agree that it is not related to this issue.